llvm/lib/WindowsDriver/MSVCPaths.cpp

```diff
@@ -475,6 +482,12 @@ bool getUniversalCRTSdkDir(vfs::FileSystem &VFS,
           Path, nullptr))
     return false;
 
+  if (WinSdkVersion) {
```

Comment:
The registry currently takes precedence here - should this not come before that? The only issue then remains is what if they specify `/winsdkdir` and `/winsdkversion`? I think that we should prefer to use the version requested against that SDK directory (or the default path if one is not specified).

Reply:
If `WinSdkDir` is already specified, it will be handled in `getWindowsSDKDirViaCommandLine()`.
`getSystemRegistryString()` will set `Path` (which we need) or return false. I think this is the right place for this.

Reply:
Ah, right - missed that the output needs both.